src/renderer/components/+preferences/preferences.tsx

```diff
@@ -196,6 +250,21 @@ export class Preferences extends React.Component {
                 themeName="lens"
               />
             </section>
+            <hr/>
+            <section>
+              <SubTitle title="Editor configuration" />
+              
+              <Tabs className="flex" scrollable={false} onChange={this.onEditorTypeChange} value={this.activeEditorType}>
+                <Tab value={EditorType.TABS} label="Tabs" data-testid="editor-tabs"/>
+                <Tab value={EditorType.DETAILS} label="Details" data-testid="editor-details"/>
+                <Tab value={EditorType.ADD_CLUSTER} label="Adding cluster" data-testid="editor-add-cluster"/>
+                <Tab value={EditorType.KUBECONFIG} label="Kubeconfig" data-testid="editor-kubeconfig"/>
```

Comment:
Tabs is a little confusing since it is actual Dock entries and not tab components.

I still think that ADD_CLUSTER and KUBECONFIG should be one setting (since they are both kubeconfigs).

And in all actual fact I think we should have settings for "narrow views" and "wide views" instead. With "narrow views" being the details page and the wizard, and "wide views" being the dock and the "add-clusters". @jakolehm WDYT?

Reply:
> I still think that ADD_CLUSTER and KUBECONFIG should be one setting (since they are both kubeconfigs).

Fixed